### ISSUE TITLE
Add Dublin Core RDF metadata export

### ISSUE THREAD

**Issue by ninsbl** [geonetwork/schema-plugins#26]
Would be great if GeoNetwok could export metadata to Dublin Core RDF (https://www.dublincore.org/specifications/dublin-core/dcmes-xml/), so it can be easily imported into Bibliography software like Zotero...

That would make it easier to cite data...

There is supposed to be a standard for mapping ISO and Dublin Core:
https://joinup.ec.europa.eu/solution/mapping-between-dublin-core-and-iso-19115-geographic-information-metadata/releases

**Comment by fxprunayre** [comment]
> into Bibliography software like Zotero...

Maybe Datacite format is supported ? https://github.com/geonetwork/core-geonetwork/pull/3212

**Comment by ninsbl** [comment]
Yes, if one retrieves a DOI for a dataset, import in Zotero is straight forward!
But it would be still helpful to have a compatible format also without requiring a DOI...

**Comment by ninsbl** [comment]
DataCite XML does not seem to be supported (yet):
https://forums.zotero.org/discussion/comment/359445#Comment_359445
